apis/v1alpha1/opampbridge_types.go

```diff
@@ -105,6 +105,8 @@ type OpAMPBridgeSpec struct {
 	// https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
 	// +optional
 	TopologySpreadConstraints []v1.TopologySpreadConstraint `json:"topologySpreadConstraints,omitempty"`
+	// PodDNSConfig defines the DNS parameters of a pod in addition to those generated from DNSPolicy.
```

Comment:
@yuriolisa should we also make it possible to configure DNSPolicy?

Reply:
@jaronoff97, actually not, because all the three possible values are contemplated [here](https://github.com/open-telemetry/opentelemetry-operator/pull/3077/files#diff-34db83fc5c513943d88ada8c831ee83c992c8250ff9e791d57ac985f73623de4) following the precedence.

Reply:
Just adding that I prefer this approach because it helps the end user avoid configuration mistakes.

Reply:
is there any other case where the user would want to set the DNSPolicy separately?

Reply:
IMO, no. It's just to offer a safe place to deliver the combination of HostNetwork, DNSPolicy, and DNSConfig.